index.html

```diff
@@ -113,6 +113,67 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card END ________  -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Daniel Lounsbury</p>
+          <p class="contact">
+            <i class="fa-brands fa-instagram"></i>
+            <a href="https://www.instagram.com/" target="_blank">daniel.loun</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com" target="_blank">BigSkyCoder2003</a>
+          </p>
+          <p class="about">I am a person</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://benddevops.org/" target="_blank" title="First Resource">Bend DevOps</a>
+              </li>
+              <li>
+                <a href="https://www.cpuid.com/softwares.html" target="_blank" title="Second Resource">Local Hardware Monitoring and Stress Testing Software</a>
+              </li>
+              <li>
+                <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ" target="_blank" title="Third resource">Good Talk</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card END ________  -->
+
```

Comment:
Looks like you accidentally duplicated another template block here which would be very confusing for future contributors (like yourself!) who want to put their card in the right place.

Can you please remove this extra template so we can get your PR merged? :D